src/test/java/com/basho/riak/client/core/util/DefaultCharsetTest.java

```diff
@@ -0,0 +1,49 @@
+package com.basho.riak.client.core.util;
+
+import java.nio.charset.Charset;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class DefaultCharsetTest
+{
```

Comment:
Ahh ok. Yeah, we should have that test here: https://github.com/basho/riak-java-client/pull/550/files#diff-09fa926e842365e4bcfc9a7cd97443f8R18